### PR TITLE
[#2042] Fix developer portal z-index and other issues

### LINKED ISSUE #2042: Projects page menu is broken on laptop sizes
**Bug Description**
When browsing on projects with 1024 width screen the top link/breadcrumb are overlapping on the menu. 

**Steps to Reproduce**
1. Go to 'https://thenewboston.com/developer/projects'
2. Click on 'hamburger icon'
4. See error.

**Expected behavior**
When the hamburger icon is clicked the mega menu will be fully shown.

**Actual behavior**
The top links and breadcrumb are overlapping with the mega menu that's why we can't see some of the text. when the width of screen is 1024.

**Screenshots/Recordings**
image

**OS and Browser**
 - Browser Chrome

**Account Number**
6a25290a1c04252563d286790194bfd341e2656bcad17e9bbf75d7a626946067

### PR DESCRIPTION
**Pull request by jamessspanggg**
Fixes #2042

### CODE REVIEW

File: src/styles/_z-index.scss [hunk 1]
```diff
@@ -19,6 +19,6 @@ $z-index-slideup-root: 150;
 
 $z-index-popover-root: 200;
 
-$z-index-dev-portal-top-links: 3;
-$z-index-dev-portal-breadcrumb: 2;
```
Comment (thesanjeevsharma) [inline src/styles/_z-index.scss:23]: this was the reason 👇 for keeping high z-index

image

Comment (thesanjeevsharma) [reply src/styles/_z-index.scss:23]: To reproduce this:
1. open any project details page.
2. test on a device with a small height, or simply open dev tools at the bottom.

Comment (jamessspanggg) [reply src/styles/_z-index.scss:23]: ah ok thanks for spotting it, i have increased the z-index range of values to the hundreds to allow more room to work with. Fixed this as well

File: src/styles/_z-index.scss [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-$z-index-slideup-root: 150;
+$z-index-slideup-root: 1500;
 
-$z-index-popover-root: 200;
+$z-index-popover-root: 2000;
 
-$z-index-dev-portal-top-links: 3;
-$z-index-dev-portal-breadcrumb: 2;
+$z-index-dev-portal-top-links: 100;
+$z-index-dev-portal-breadcrumb: 100;
```
Comment (thesanjeevsharma) [inline src/styles/_z-index.scss:23]: @jamessspanggg let's make this 99.

image

File: src/styles/_z-index.scss [hunk 3]
```diff
@@ -1,24 +1,24 @@
 $z-index-webmap: 0;
 
-$z-index-project-header: 1;
+$z-index-project-header: 99;
```
Comment (thesanjeevsharma) [inline src/styles/_z-index.scss:3]: let's make this 98

File: src/styles/_z-index.scss (hunk 2 above)
Comment (thesanjeevsharma) [reply src/styles/_z-index.scss]: nvm, I'll did it! 👍

File: src/styles/_z-index.scss (hunk 3 above)
Comment (thesanjeevsharma) [reply src/styles/_z-index.scss]: done

Comment (jamessspanggg) [reply src/styles/_z-index.scss]: done, thanks

**Review by thesanjeevsharma — APPROVED** [review]

**Pull request merged**